cordova-lib/spec-plugman/install.spec.js

```diff
@@ -345,13 +344,14 @@ describe('install', function() {
 
         it('should not check custom engine version that is not supported for platform', function() {
             var spy = spyOn(semver, 'satisfies').and.returnValue(true);
-            runs(function() {
-                installPromise( install('blackberry10', project, plugins['com.cordova.engine']) );
-            });
-            waitsFor(function() { return done; }, 'install promise never resolved', 200);
-            runs(function() {
+            install('blackberry10', project, plugins['com.cordova.engine']).then(function(done) {
+                expect(false).toBe(true);
+                done();
+            },
+            function err(errMsg) {
                 expect(spy).not.toHaveBeenCalledWith('','>=3.0.0');
```

Comment:
Thanks @stevengill ... It looks like in some of the other tests, it is going right to fail on purpose? I can rewrite the test like this 

```
        it('should not check custom engine version that is not supported for platform', function() {
            var spy = spyOn(semver, 'satisfies').and.returnValue(true);

            install('blackberry10', project, plugins['com.cordova.engine'])
            .then(function() {
            }).fail(function(error) {
                expect(spy).not.toHaveBeenCalledWith('','>=3.0.0');
            }).fin (done); 
}); ```

Is this better? It still goes right to the fail.